opentracing-spring-rabbitmq/src/main/java/io/opentracing/contrib/spring/rabbitmq/RabbitMqBeanPostProcessor.java

```diff
@@ -35,11 +35,6 @@
   @Override
   public Object postProcessBeforeInitialization(Object bean, String beanName)
```

Comment:
Due to the following change in spring-rabbit-2.1.4:
https://github.com/spring-projects/spring-amqp/pull/815

the interceptors now have to be registered in the BeforeInitialization method as opposed to the AfterInitialization method in order for the bean's afterPropertiesSet() to correctly detect the presence of the interceptor in the adviceChain and create the proxy in SimpleMessageListenerContainer.